src/Collapsable.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react/cjs/react.production.min"
+
+export const Collapsable = () => {
```

Comment:
The collapse functionality needs to be implemented in Flowers component.
Here you're toggling this text " "A magnificent variety, bearing large, many petalled rosette blooms. They are a striking shade of deep pink, the outer petals of each bloom paling slightly over time. The beauty of the blooms is enhanced by a wonderful, strong fruity fragrance."

how do you know that the text won't be something else?


but if you make the description in side the Flowers component collapsible then you're sorted

Reply:
Thanks 
I will check this later